src/consts/gasTankFeeTokens.ts

```diff
@@ -39,7 +39,8 @@ export default [
     symbol: 'aeth',
     networkId: 'arbitrum',
     decimals: 18,
-    icon: 'https://assets.coingecko.com/coins/images/279/large/ethereum.png'
+    icon: 'https://assets.coingecko.com/coins/images/279/large/ethereum.png',
+    isAave: true
```

Comment:
This one is not an AAVE token, it's the native ETH on Arbitrum. For reasons I don't know we've named it AETH.  
I guess you've made it isAave: true because you want to hide it from the error message.  
In that case, we can rename this to "hiddenOnError" for now. Just a proposal